client/app/views/popover_screens/guests.coffee

```diff
@@ -158,31 +183,79 @@ module.exports = class GuestPopoverScreen extends EventPopoverScreenView
     # Handle guest addition. `userInfo` is passed when called by the typeahead.
     onNewGuest: (userInfo = null) ->
 
+        # Autocomplete was used.
         if userInfo? and typeof(userInfo) is "string"
-            [email, contactID] = userInfo.split(';')
+            [channel, contactID] = userInfo.split(';')
+        # Field was entered manually.
```

Comment:
I think relying on a string split for knowing the source of the contact is an anti-pattern, but I looked into the typeahead plugin and it will be really painful do proceed in another way. So let's go with that :)

(Ideally, we should have an object in parameter for `onNewGuest`. This object should be something like :
'''javascript
{
    contact: fetchedContact
    source: 'cozy'
}
'''
I think we should keep this idea in the backlog, but it's not the right time now.